### PR TITLE
Revamp Overview again according to new design.

### PR DESCRIPTION
**Pull request by yuchenshi**
image

### CODE REVIEW

**Comment by samtstern** [comment]
I don't know where the string comes from but it says "local enviornment" not "local environment"

We could leave that in as PR bait though :-)

File: src/components/Home/index.tsx [hunk 1]
```diff
@@ -19,13 +19,54 @@ import { connect } from 'react-redux';
...
+        <Typography use="headline6" tag="h3">
+          {name}
+        </Typography>
+        <Typography use="subtitle2" tag="div">
+          Status
+        </Typography>
+        <Typography use="headline6" tag="div">
+          On
+        </Typography>
+        <br />
```
Comment (tjlav5) [inline src/components/Home/index.tsx]: nit: don't use breaks for styling

**Comment by yuchenshi** [comment]
> I don't know where the string comes from but it says "local enviornment" not "local environment"
> 
> We could leave that in as PR bait though :-)

Not introduced by this PR, but I've fixed it anyway.

File: src/components/Home/index.tsx (hunk 1 above)
Comment (yuchenshi) [reply src/components/Home/index.tsx]: Sure, no breaks now.

**Pull request merged**